frontend/src/pages/org/settings.ts

```diff
@@ -220,16 +220,17 @@ export class OrgSettings extends LiteElement {
                 const input = e.target as SlInput;
                 this.slugValue = input.value;
               }}
-            >
-            </sl-input>
+            ></sl-input>
           </div>
 
           <div class="col-span-5 md:col-span-2 flex gap-2 pt-6">
             <div class="text-base">
               <sl-icon name="info-circle"></sl-icon>
             </div>
             <div class="mt-0.5 text-xs text-neutral-500">
-              ${msg("Unique URL for this organization.")}
+              ${msg(
+                "Customize your org web address for accessing Browsertrix Cloud."
```

Comment:
```suggestion
                "Customize your org's web address for accessing Browsertrix Cloud."
```

Reply:
i think it would make more sense if 'organization's'

Reply:
Updated suggestion